### ISSUE TITLE
With version 3.0.4 and above, using glide:3.7.0 results in a crash error with the error below

### ISSUE THREAD

**Issue by OpheliaKing** [coolishbee/universal-sdk-unity#41]
04-04 19:34:14.513 26376 26376 Error AndroidRuntime java.lang.NoSuchMethodError: No virtual method load(Ljava/lang/String;)Lcom/bumptech/glide/DrawableTypeRequest; in class Lcom/bumptech/glide/RequestManager; or its super classes (declaration of 'com.bumptech.glide.RequestManager' appears in /data/app/~~LfHRg8THlRoY8Ss-awztoQ==/com.test.test-WldYxIGReQ4_hH16d0vTQQ==/base.apk)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at com.navercorp.nng.android.glideadapter.GlideAdapter.loadImage(GlideAdapter.java:16)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at a.b.a.a.b.m0.b.a(SourceFile:4)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at a.b.a.a.b.m0.b$a.a(SourceFile:3)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at a.b.a.a.b.i0.b$a.a(Unknown Source:25)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at a.b.a.a.b.i0.b$a$$ExternalSyntheticLambda2.run(Unknown Source:4)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at android.os.Handler.handleCallback(Handler.java:942)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at android.os.Handler.dispatchMessage(Handler.java:99)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at android.os.Looper.loopOnce(Looper.java:226)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at android.os.Looper.loop(Looper.java:313)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at android.app.ActivityThread.main(ActivityThread.java:8762)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at java.lang.reflect.Method.invoke(Native Method)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
04-04 19:34:14.513 26376 26376 Error AndroidRuntime 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)

**Comment by OpheliaKing** [comment]
There is the same problem when you change the version of Glide used by the project to 4.15.0

**Comment by coolishbee** [comment]
기존에 사용하던 배포용 도구 sonatype nexus 가 레거시 서비스를 중단되여 급하게 jitpack 으로 변경하였습니다.
현재 안드로이드에서는 테스트가 되었지만 유니티에서는 테스트 진행중이라 완료되면 가이드를 제공하겠습니다.

## 안드로이드 환경에서의 가이드
Add it in your root build.gradle at the end of repositories:
```
dependencyResolutionManagement {
	repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
	repositories {
		google()
		mavenCentral()
		maven { url 'https://jitpack.io' }
	}
}
```
```
dependencies {
    implementation 'com.github.coolishbee:universal-sdk-android:3.0.9'
}
```

**그리고 이슈를 남겨주실때는 사용자 환경에 대해서 남겨주시면 정말 감사하겠습니다**

**Comment by OpheliaKing** [comment]
It's settled. Thank you

**Comment by coolishbee** [comment]
## Guide in Unity editor

- Unity editor version: 2022.3.23f1
- Target API Level: Android 13.0 (API Level 33)
- Platform you are using the Android Studio on: Mac

### Step 1. Add it in your settingsTemplate.gradle in project:
```
dependencyResolutionManagement {
    repositoriesMode.set(RepositoriesMode.PREFER_SETTINGS)
    repositories {
        **ARTIFACTORYREPOSITORY**
        google()
        mavenCentral()
        maven { url 'https://jitpack.io' }
        flatDir {
            dirs "${project(':unityLibrary').projectDir}/libs"
        }
    }
}
```
### Step 2. Add the dependency
gradle:
```
dependencies {
    implementation 'com.github.coolishbee:universal-sdk-android:3.0.9'
}
```
dependencies.xml
```
<dependencies>    
    <androidPackages>
        <androidPackage spec="com.github.coolishbee:universal-sdk-android:3.0.9"/>
    </androidPackages>
</dependencies>
```

> It will be released on the Asset Store soon.

**Comment by coolishbee** [comment]
Released as unity sdk 1.2.10
